app/addons/documents/doc-editor/actions.js

```diff
@@ -79,33 +80,24 @@ function hideDeleteDocModal () {
 }
 
 function deleteDoc (doc) {
-  var databaseName = doc.database.safeID();
-  var query = '?rev=' + doc.get('_rev');
-
-  $.ajax({
-    url: FauxtonAPI.urls('document', 'server', databaseName, doc.safeID(), query),
-    type: 'DELETE',
-    headers: {
-      'Accept': 'application/json',
-      'Content-Type': 'application/json',
-    },
-    xhrFields: {
-      withCredentials: true
```

Comment:
I think you have to include  something similar to that in fetch. I think it's:   credentials: 'include'

Reply:
The functions in `core/ajax` all use `credentials: 'include'`